bottlecap/src/config.rs

```diff
@@ -120,10 +150,9 @@ pub enum ConfigError {
 
 pub fn get_config(config_directory: &Path) -> Result<Config, ConfigError> {
     let path = config_directory.join("datadog.yaml");
-    let figment = Figment::new()
-        .merge(Yaml::file(path))
-        .merge(Env::prefixed("DATADOG_"))
-        .merge(Env::prefixed("DD_"));
+    let figment = Figment::new().merge(Yaml::file(path));
+    // .merge(Env::prefixed("DATADOG_"))
```

Comment:
need these

Reply:
Oops, my bad.

Reply:
Addressed in [ce37057](https://github.com/DataDog/datadog-lambda-extension/pull/226/commits/ce370575f00e85c82d4430597d596433e020f686)